topdown/crypto.go

```diff
@@ -190,6 +193,42 @@ func builtinCryptoSha256(a ast.Value) (ast.Value, error) {
 	return hashHelper(a, func(s ast.String) string { return fmt.Sprintf("%x", sha256.Sum256([]byte(s))) })
 }
 
+func hmacHelper(args []*ast.Term, iter func(*ast.Term) error, h func() hash.Hash) error {
```

Comment:
💭  Why `h func() hash.Hash` and not `h hash.Hash`? 💡 It's the `hmac.New()` interface that makes this the right choice. Interesting.